### DISCUSSION TITLE
Passing components as Props

### DISCUSSION THREAD

**Discussion by seb-jean** [livewire/livewire#8608] [1. Help] [answered]
Hello,

I would like to know if it is possible to pass Components as Props. Example with Headless UI from Tailwind:

`<Headless.MenuButton as={SidebarItem} />`

**Comment by joshhanley** [comment, marked as answer]
@seb-jean no, but we do have dynamic components, so you could just pass the name of a component and use that with a dynamic component to load it. See https://livewire.laravel.com/docs/nesting#dynamic-child-components

Hope that helps!
